marked-element.html

```diff
@@ -258,8 +285,26 @@
       }, null);
 
       return lines.map(function(l) { return l.substr(indent); }).join('\n');
-    }
+    },
 
+    _request: function(url) {
+      var xhr = this.xhr;
+
+      if (xhr.readyState > 0) return null;
+
+      xhr.addEventListener('loadend', function (e) {
+        var status = this.xhr.status || 0;
+        // Note: if we are using the file:// protocol, the status code will be 0
+        // for all outcomes (successful or otherwise).
+        if (status === 0 || (status >= 200 && status < 300)) {
+          this.markdown = e.target.response;
+        }
```

Comment:
Can we support the fail scenario as well? Something as simple as `this.markdown = 'text failed to load';` would add a lot of value.